spec/ruby/fixtures/constants.rb

```diff
@@ -70,6 +70,14 @@ module ModuleD
     CS_CONST10 = :const10_8
   end
 
+  module ProtectedConstMissing
+    protected
```

Comment:
Why not `private`?

Reply:
`protected` is the real world case I encountered.

Reply:
It really just has to be not public.